test-files/ui/features/highlight_features.feature

```diff
@@ -0,0 +1,51 @@
+Feature: Highlight features on click
+
+    Scenario: Start Hootenanny
+        Given I am on Hootenanny
+        And I resize the window
+        And I click Get Started
+
+    Scenario: Add Data
+        And I press "Add Reference Dataset"
+        And I click the "AllDataTypesACucumber" Dataset
+        And I press "Add Layer"
+        Then I wait 15 "seconds" to see "span.strong" element with text "AllDataTypesACucumber"
+        And I press "Add Secondary Dataset"
+        And I click the "AllDataTypesBCucumber" Dataset
+        And I press "Add Layer"
+        Then I wait 15 "seconds" to see "span.strong" element with text "AllDataTypesBCucumber"
+        Then I should see "Conflate"
+        And I press "Conflate"
+        And I fill "saveAs" input with "AllDataTypes_Highlight"
```

Comment:
Please add the Cucumber suffix to your dataset name so that `make clean` will remove it from the db.